src/utils.py

```diff
@@ -30,3 +30,43 @@ def get_label_distribution(all_labels: list) -> dict:
     labels = sorted(list(set(all_labels)))
     label_counts = {label: all_labels.count(label) for label in labels}
     return label_counts
+
+def listToString(list: list) -> str:
+    """ Converts a list into a str
+
+    Args:
+      list (list): The list to convert
+
+    Returns:
+      str made from elements of the list.
+
+      e.g. "This was a list" 
+    """
+    string = " "
+    return (string.join(list))
+
+def read_data(path: str) -> (list, list):
```

Comment:
Don't we already have `read_documents` above that does exactly this?

Reply:
Oh ya, sorry. I got caught up and made my own version of what already existed.